site/documentation/content/api/Tenant-API.md

```diff
@@ -152,36 +152,37 @@ The JSON structure below contains example information for tenant `TEST_TENANT`.
+If the *trusted-ca* property is provided, then it MUST contain a non-empty array of JSON objects, were each JSON object represents a single trusted CA. The table below provides an overview of the members defined in a JSON object contained in the *trust-ca* property:
 
 | Name                     | Mandatory  | Type          | Default Value | Description |
 | :------------------------| :--------: | :------------ | :------------ | :---------- |
-| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
-| *cert*                   | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root X.509 certificate. |
+| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). The subject DN MUST be unique across tenants. |
 | *public-key*             | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
-| *not-before*             | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid before the given date. |
-| *not-after*              | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid after the given date. |
+| *not-before*             | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid before the given date. The date format is defined by [RFC 5280](https://www.ietf.org/rfc/rfc5280.txt). |
+| *not-after*              | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid after the given date. The date format is defined by [RFC 5280](https://www.ietf.org/rfc/rfc5280.txt).|
 | *algorithm*              | *no*       | *string*      | `RSA`        | The name of the public key algorithm. Supported values are `RSA` and `EC`. This property is ignored if the *cert* property is used to store a certificate. |
 
 * The *subject-dn* MUST be unique among all registered tenants.
 * Either the *cert* or the *public-key* MUST be set.
-* If the *cert* property is used to store a certificate then the *not-before*, *not-after* and *algorithm* properties are ignored.
 
 **Examples**
 
-Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT` configured with two trusted certificate authorities.
+Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT`. The tenant is configured with two valid trusted certificates issued by the same issuer and having overlapping validity periods.
```

Comment:
we don't know anything about the issuer. But the public keys are associated with the same *subject*

Reply:
thanks. I was confusing this with the issuer of client certificates which (usually) matches the subject-dn of the trusted CA.

Reply:
fixed in  4380d8c